### ISSUE TITLE
Public pretrained models

### ISSUE THREAD

**Issue by minh132** [Valfride/lpr-rsr-ext#1]
Can you provide pretrained model

**Comment by pseudo-usama** [comment]
Did you train this model and test it?

**Comment by valfride** [comment]
@pseudo-usama yes, this model was trained and tested, refer to the paper for more informations about the methodology used.

**Comment by valfride** [comment]
@minh132 Could you provide your e-mail, please? so I can send a link to download the models.

**Comment by pseudo-usama** [comment]
Thank you! It's pseudo.usama@gmail.com
Also can't you just make model public on google drive or even here on github?

**Comment by ghost** [comment]
> @minh132 Could you provide your e-mail, please? so I can send a link to download the models.

I need the pretrained models ,too. Could you send me a link to my e-mail:  lxning530@126.com ? Thanks  a lot.

**Comment by pseudo-usama** [comment]
> Thank you! It's [pseudo.usama@gmail.com](mailto:pseudo.usama@gmail.com) Also can't you just make model public on google drive or even here on github?

@Valfride I didn't get any email. Can you please confirm?

**Comment by nerdbamboo** [comment]
> @minh132 Could you provide your e-mail, please? so I can send a link to download the models.

Thank you for your great work! I also need a pre-trained model to test my custom dataset. Could you please provide a link?
My email is [22101070@handong.ac.kr ](url). Thank you!

**Comment by lehoangHUST** [comment]
I need the pretrained models, too. Could you send me a link to my e-mail: lehuyhoang30122000@gmail.com ? 
Thanks you very much

**Comment by WJM0805** [comment]
Thank for you works.
Coud you provide the training weights for your OCR model. 
My email is 2501780961@qq.com.

**Comment by lisasoohyun** [comment]
Thank for you works.
Coud you provide the training weights for your OCR model.
My email is tngusdldlt@naver.com

**Comment by oron-sinaa** [comment]
Hey, thanks for the great work! I'd be grateful if you'll share the training weights to aanis.noor@staqu.com.

**Comment by CasvalRay** [comment]
Thank for you works.
I need the pretrained models and  OCR model, too. Could you send me a link to my e-mail: [874752783@qq.com]
Thanks you very much
